static/app/components/modals/inviteMembersModal/index.tsx

```diff
@@ -65,6 +71,22 @@ function InviteMembersModal({
     );
   }
 
+  const errorAlert = error ? (
+    <Alert type="error" showIcon>
+      {error}
+    </Alert>
+  ) : null;
+
+  const inviteMessage = willInvite ? (
+    <Subtext>{t('Invite unlimited new members to join your organization.')}</Subtext>
+  ) : (
+    <Alert type="warning" showIcon>
+      {t(
+        'You can’t invite users directly, but we’ll forward your request to an org owner or manager for approval.'
+      )}
+    </Alert>
+  );
```

Comment:
nit these could be separate components with props. the advantage of having this defined as a separate component is how JS will run this code.

in practice what will happen is that every time there's a state change or button click, we'll redefine this component. if we pull it out as a separate component - we will only need to recompute these if the property changes.